jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1532,34 +1532,34 @@ ecma_builtin_typedarray_prototype_sort (ecma_value_t this_arg, /**< this argumen
   JERRY_ASSERT (buffer_index == info.length);
 
   const ecma_builtin_helper_sort_compare_fn_t sort_cb = &ecma_builtin_typedarray_prototype_sort_compare_helper;
-  ECMA_TRY_CATCH (sort_value,
-                  ecma_builtin_helper_array_heap_sort_helper (values_buffer,
-                                                              (uint32_t) (info.length - 1),
-                                                              compare_func,
-                                                              sort_cb),
-                  ret_value);
-  ECMA_FINALIZE (sort_value);
 
-  ecma_typedarray_setter_fn_t typedarray_setter_cb = ecma_get_typedarray_setter_fn (info.id);
+  ecma_value_t sort_value = ecma_builtin_helper_array_heap_sort_helper (values_buffer,
```

Comment:
If everything was ok during the sort what is the expected `sort_value`? Maybe we could have an assert for it?